src/main/java/org/mockbukkit/mockbukkit/world/WorldMock.java

```diff
@@ -554,6 +554,23 @@ else if (c.y <= grassHeight)
 		return block;
 	}
 
+	public @NotNull void setBlock(@NotNull BlockMock block)
```

Comment:
Please remove the "NonNull" annotation from the return type since it's void

Reply:
Removed.